exo-go/src/types/secrets.go

```diff
@@ -0,0 +1,49 @@
+package types
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// Secrets map contains maps from secret keys to values
+type Secrets map[string]string
+
+// NewSecrets creats a Secrets map from a .tfvars string
+// a="b"
+// c="d" ->
+// {a:b, c:d}
```

Comment:
Is this comment needed now that we have the tests for it? I also don't think its very clear. I would say the example needs to be written as:

```
Input: `a="b"\nc="d"`
Output: Secrets{"a": "b", "c": "d"}
```